app/screens/create_direct_message/selected_users/index.tsx

```diff
@@ -1,52 +1,92 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useMemo} from 'react';
-import {View} from 'react-native';
+import React, {useCallback, useMemo} from 'react';
+import {ScrollView, View} from 'react-native';
 
 import FormattedText from '@components/formatted_text';
 import {useTheme} from '@context/theme';
+import Button from '@screens/bottom_sheet/button';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 
-import SelectedUser from './selected_user';
+import SelectedUser, {UserChipBottomMargin, UserChipHeight} from './selected_user';
 
 type Props = {
 
     /*
-     * An object mapping user ids to a falsey value indicating whether or not they've been selected.
+     * An object mapping user ids to a falsey value indicating whether or not they have been selected.
      */
     selectedIds: {[id: string]: UserProfile};
 
     /*
-         * How to display the names of users.
-         */
+     * How to display the names of users.
+     */
     teammateNameDisplay: string;
 
     /*
-         * The number of users that will be selected when we start to display a message indicating
-         * the remaining number of users that can be selected.
-         */
+     * The number of users that will be selected when we start to display a message indicating
+     * the remaining number of users that can be selected.
+     */
     warnCount: number;
 
     /*
-         * The maximum number of users that can be selected.
-         */
+     * The maximum number of users that can be selected.
+     */
     maxCount: number;
 
     /*
-         * A handler function that will deselect a user when clicked on.
-         */
+     * A handler function that will deselect a user when clicked on.
+     */
+    onPress?: (selectedId?: {[id: string]: boolean}) => void;
```

Comment:
Does it make sense for this to be undefined?

Reply:
If not, we can remove the usecallback on line 115.

Reply:
I haven't went through all the cases that will use this screen, but don't foresee one that won't have the button present.  I'll make `onPress` required and add back as optional if that changes later.